client/src/components/questions/AnswerModal.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled, { createGlobalStyle } from 'styled-components';
+
+const StyledBackground = styled.div`
+  background: rgba(0, 0, 0, .5);
+  position: fixed;
+  top: 0; left: 0;
+  width: 100vw; height: 100vw;
```

Comment:
same? Is there a reason you put these on the same line?

Reply:
it was how they did it on a website I looked at. I think it makes sense given how short the lines are but you are probably right.